pkg/scheduler/apis/config/types.go

```diff
@@ -90,11 +90,6 @@ type KubeSchedulerConfiguration struct {
 	// nodes will be scored.
 	PercentageOfNodesToScore int32
 
-	// Duration to wait for a binding operation to complete before timing out
-	// Value must be non-negative integer. The value zero indicates no waiting.
-	// If this value is nil, the default value will be used.
-	BindTimeoutSeconds int64
```

Comment:
Let's proceed with this removal, but not the one for v1alpha2 folder. This way, we can merge this PR without worrying about v1alpha3/v1beta1, but being prepared for it.

Reply:
Lets separate the issue of removing the parameter from migrating functionality to plugins, injecting the args as a PluginConfig temporarily in factory isn't a big issue.